database/migrations/2024_02_22_120000_add_all_users_to_announcements_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void
+    {
+        Schema::table('announcements', function (Blueprint $table) {
+            $table->boolean('all_users')
+                ->default(false)
+                ->after('users');
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('announcements', function (Blueprint $table) {
+            //
```

Comment:
please drop all_users column in this line

Reply:
@rupadana is it okay?

Reply:
is it okay now?

Reply:
sorry to letting this request still opened

i think this problem already fixed on this PR #22